### PR TITLE
Add Possibility to permanantly delete node + restore multiple nodes

### PR DESCRIPTION
**Pull request by lilian-houdelet**

### CODE REVIEW

**Review by AbdelHedhili — CHANGES_REQUESTED** [review]
There is a bug : 
 - create a subtree of at least 2 nodes
 - delete that subtree
 - delete it permanently 
 - go in the restore/delete popup and you will see that only the first node was permanently deleted and not the whole subtree

File: src/translations/en.json [hunk 1]
```diff
@@ -654,7 +654,6 @@
     "BusBarBus": "Bus bar section / bus",
     "Setpoints": "Setpoints",
     "Connectivity": "Connectivity",
-    "Connected": "Connected",
```
[the remark this answers is not in the document]
Comment (lilian-houdelet) [reply src/translations/en.json:657]: Duplicate key, if I'm not wrong

[the remark this answers is not in the document]
Comment (AbdelHedhili) [reply src/translations/en.json:657]: Indeed, good catch, however we avoid doing changes that isn't related to the feature/bug you are working on in the same PR. Move this into another PR please.

**Review by AbdelHedhili — APPROVED** [review]

**Review by souissimai — APPROVED** [review]
seems ok

**Pull request merged**